Config/Cessna/options_magnetometer.h

```diff
@@ -21,7 +21,7 @@
 // To do: select magnetometer type, set MAGNETICDECLINATION,
 // and select orientation of the magnetometer, and remove the next 3 lines.
 #if (MAG_YAW_DRIFT == 1 && HILSIM != 1)
-#warning "Check magnetometer options."
+//#warning "Check magnetometer options."

```

Comment:
I agree, and have never much liked the plethora of warnings this can produce on the default options build. If we choose to keep this warning, I'd be in favour of at least moving it into the magnetometer source file and then at least it'll only chime in once. That said, for the moment I'd like to leave such a change to another PR.